examples/Whisper_transcribe_device_microphone.ipynb

```diff
@@ -0,0 +1,500 @@
QoGZQBEOC1yZXkc1Edu4JYG14JqrhyaPRizDJxuN4KisCqplMSwb9HsliPsza4vJDfOuBtLF8yoOudwwEDmbKivKkbk7uYxl9sugWiyUeRiCe2mGDuEKYCigUIuAdfyKGIu/U4mzf5RlxTCOgUydNImJR8owhFd58RiSkMHILEvKfUuDBgwSccqfYSOvQ2iPjNHCK8uDkFvMzzC7h4sMz5DQ64w3SYoRtHwM5HiloJqlx2C/APC3YHus+ph1dOsMt1DRIG0M0DCOiHtEsZvQKL2nfAwLqNt4KYmFLPnC42t9+P86zNBbGdqJkzDYyFaaull2xpErsY7ruSGDq4B60AHMsGbsoPP8uG4pgO+sQp/JksgLz+zMvrUgqYL3wJ3qu2VbxrVofccrw67Md2BHEL4QDWSbgeAJstuMkTzNCZuLxqi8Zuwb193LJa0I1KnsPcGQusaHl4QyCSv8toZ9ttB6qEmsZjI6xVtJPHZrt8MMoffap5bok3HtqLgMuKorgYV9ukx9nXNLhDi51epZaUr9CqfMlNtoCwU8Dgfy17YAtDt8FhWHRLPAyng3JTQreWp6ap83q6LADIX8b6P7wVNocB68qkOiQt13HNRwiJ+XDM99hr0wxbxZrbmQzG2JmHX/CCrm0Y8IYX4l2It1J9zUGtHAmM7jZ3Ei46wMgdiNLJjKfL7Q6A2Sl6OPK87Ai6JyQm8uK8M+vX2zKdvSMMFhOhJkPJB3WrrZkOB8bKyS8adBgn72uWVZvbKpvdq9CI7SWqiHq7CKws08FK+OiRW+HQ0+3OAg0Qz4KsXBS47Yh7x4PIrdEAp/OYoJLNPEwr9AGOPRGpBBrrAWebtVHRHXXbOUKdz6fRA2u7hYSJzgqq1K4+BUhpP+2+Tp6h/IoLLg0Mk4CrKhKrwGPoLVLRCbILVZXOGc7Q016sXErD8BS8sSAbpvziPdGiisTc3qq7tz569tHdkHQcKd3NLbXIpM/63aiY5wbW5lMlfa9ywWu9oW6Srd+wN5VH7OduyOqb3OrCfUyPwMiJjg+Hq4vsCslguLiNjktkm8XwN4ZaqmSBygWevj1fqW3M3cQumk9OwzP/ulcC6ltQC/jvx54FCav3DLyXohDZqpcdq4sfzmRz3HmW4RQVrQZRsMc6gMWgqIbnzXh/Qel0DYsZ2aIWqTy1plxayNxxyF6Pt6pArpG47Wb0zUa+5zrpumh87h5GbIhjnl8S20ZEG+bJEMvHDTgrg9W50lupm+G6e0TYidKquasWwKlvCbk8tsaq2aHkusUU3jFTgW43jNc9mqjSedWo1lZyupWwy+ryfo77kQDf9+CwqXytNcnXHxoebutwmJzkkCwCwvGSRczeC4DQlO3Qg4pQ6PJnVNjSU/pOru01ZaGaWAeGHIowOOk6z9oTRvZUMvt3gZxtl24Zow7fX7wfxuabu834Fd7ANfF6lg0QOYOjgeP9Nyyjldj/7ZbhuMv94B7JIpEvqagEiXm/rsELlgsSKOhC/60oSb3Ss6uROB7a9rxf1a60NdoRxtzS5qg7NB+d7R5f8c4KmOaTnNJqGarU89l2fr+Fi7DPHbCtmsDIItEteHZaKw8mZqp4suC1+/m4I948c6nk3C1y4utsTO0a+t+WEioa5xFK4XJckjGnkDkcZwydQYu0jvItf/f2Vj3gxkKMHhHcn7aMi1INJxHLaFu2X8KgxWW0a96SoGX6i0mIEcqrMgd/6wiP+4DxC+eAEgSHCWtVcJSYGiNgpXwWcGs3maVrAiwWfIkBlLFgoSJ5CPLBYE8EqgsFYLP4Fk2dKlKmUDAXRUSFKaxl4SO73k2bKWM4sJSUL0dZChplu8oMXaZmohRqWjpu0kCbRkT4y4dr6MSRJWK4LEjJL82PLXM1yrKo0EkBFnUKchyb5y1YyXK4oqUSWTGRZVpZar5q7chlTYr6bUGiYt+MhTL2fH8g6uKwxvS6g6Jw9+NdTzZ9ChRY8mXToYaWyXUqtO/TlbadixZc+mXds2/2lRj2Ddxj13FCuC3MiS9dYpNMvSyD1P1Wu8pESD0Etyo16NFG/br4OzTa7TdfSrPIdqH/79eHFrcDGPB93c/XXXwJYrxERN/Xu9552H51zdczfvYoNvufLEGc+69kAq7zkGOVPQPLgSXI+9Bturb8DOKOSOQUy4A3C+nkIUUTzsZuPEttVSvKREFlt08cXsUCllNxh7K3FAAtsT8MEaW6SqwuMEvI9EHbsrcsMeHdQwSQZNye48Jj0ZDZQC5wpSvxNpw3ErJ49kkcqhMlmQSTIhrE1FNFtDskw2RcuyTdhyo4W+Eeu0k8s789TzTjhv5LG2pwjS5DaX/MSPsD0TLf8UREUbdfRRSBMdMtIJ+yRUQkrttPS2NFfMcVM2/wQP1ONm7DNTVFNVdVVNBV1zyhdZfRRTWWu19VZcOYOzufhwXZRUFVFjbdjVSLVU1FGN9cwVwXb19VloowXsVWU18zVOabPVttYpa7I2126vPdaqTj0VrVwVf6mWzDfX3XRbeONt111667X3XnxlC9RY6t5tzFJigx0NXXOpzbdFTZyb92De5HX44TyDYnhiiiuuN8O2wALy1Lw6lhbYcoVFM8eFLb7tGWVEYQ5HkxmF+GWYI255ZpprZtfYl2zmNOSBA2bNSp1Jiwi0ZeJiuU1Tv4x5aaabdnrPoKOWutd42U3/DzSfBQZAtaG0xnRqjDDyJSwzo/4VbLTTVnttttt+Msqe0/SMa5II/hRtXTS7d2VU3bb4aCzF9dtBZJP85NjBE9+xcIBHHrPrgOdD25WCcjnavsov0vnsJRXHV9RDV1Wc8UqtVnte97Q02HMjBZXSRcBDG3ZBzD/j2Zi2mxkTzLZAXQsAWEJvVFDI8GJ94j93ScxWfRmveOGSywSMeemiVO4qM49K7vrj9U3WxVKwKY3uJuGrnaCsZw8QbGm8cn3Z6KPUTfShYum+5Qu5NYiVseNDm66+bCUX0OiFKuQTKt/FCnoHKsg3
```

Comment:
ditto

Reply:
Hi @ibigio! Thanks for the response and review. 

Unfortunately, the ipynb files render as json and displays only line 3 - 6. 

Do you have a fix for this? It seems like a lot of people experience this too? https://github.com/orgs/community/discussions/12959